blocks/pocketguide/pocketguide.css

```diff
@@ -0,0 +1,41 @@
+.pocketguide {
+    text-align:center;
+    position:relative;
+    padding-top:136px;
+    border-radius:10px;
+    border:1px solid #7f5732;
+    margin:100px 15px;
+    min-width: 200px;
+}
+
+.pocketguide picture{
+    height:190.8px;
+    width:127.4px;
+    right:calc(50% - 63.7px);
+    top:-60px;
+    position:absolute;
+    transform:rotate(13deg)
+}
+
+
```

Comment:
the BP in the middle at 1000 is basically triggered by font size changes only, so the pocketguide does not really care at this point in time. when reaching 1152, there's more to do, not just font sizes.